lib/llvm-backend/src/state.rs

```diff
@@ -405,4 +405,44 @@ impl<'ctx> State<'ctx> {
             if_else_state: IfElseState::If,
         });
     }
+
+    pub fn rauw_block(&mut self, from: BasicBlock<'ctx>, to: BasicBlock<'ctx>) {
+        for ref mut frame in &mut self.control_stack {
+            match frame {
+                ControlFrame::Block { ref mut next, .. } => {
+                    if *next == from {
+                        *next = to
+                    };
```

Comment:
I think the `;` after the `if` block is unnecessary (as everywhere in this function).